python/pyspark/sql/dataframe.py

```diff
@@ -1181,18 +1181,26 @@ def approxQuantile(self, col, probabilities, relativeError):
         Space-efficient Online Computation of Quantile Summaries]]
         by Greenwald and Khanna.
 
-        :param col: the name of the numerical column
+        :param cols: str, list.
+            The name(s) of the numerical column(s). Can be a string of the name
```

Comment:
I think we can simplify this comment to: `Can be a single column name, or a list of names for multiple columns`. I think it's clear from the specified types that it's a string name or a list of string names.

(we mention in the method doc that it operates on numerical columns, we don't need to repeat that).